committee.rst

```diff
@@ -0,0 +1,152 @@
+  may choose, however, to use public channels, in egregious circumstances.)
+  The committee does not need to weigh in during this process, though the
+  expectation is that if the co-chairs abuse their power, the committee will
+  vote them out of their chairships.
+
+* The committee has one secretary. The secretary is the first stop for keeping
+  our repository up-to-date and running our selection process.
+
+* The committee has one nudger. The nudger's job is to make sure other
+  committee members are fulfilling their duties by nudging them (often
+  in public) to do so.
+
+The co-chairs and secretary
+positions remain with an individual until they decide to relinquish the
+role, though a majority vote among the committee can strip a member of
+their co-chair/secretary position.
+
+The nudger position rotates among members every six months.
+In the event that the nudger ends up needing nudging (by any other member
+of the committee), the nudger should step down and volunteer to do a better
+job in some future six months; in the event that they still struggle to
+stay on top of their nudging duties, it may be a sign that they are too
+busy for committee work and should step down.
+
+The secretary is in charge of managing who the nudger is, by creating
+a rotation for this position.
+
+Guidelines for Communication
+----------------------------
+
+All members of the Steering Committee agree to abide by the
+`Guidelines for Respectful Communication <https://github.com/ghc-proposals/ghc-proposals/blob/master/GRC.rst>`_. As the guidelines document
+describes, if one of us errs in our communication, please
+opt to bring up the matter with that individual directly.
+If that is not feasible, then please bring it up with the
+(co-)chair of the committee. Everyone should feel welcomed
+to contribute to the growth of GHC and Haskell, and we eagerly
+want to know when our actions cause someone not to feel that
+way.
+
+Method of Communication
+-----------------------
+
+Most official committee business takes place via the
+`ghc-steering-commitee <https://mail.haskell.org/mailman/listinfo/ghc-steering-committee>`_ mailing list. The archives of this list are public, and
+any member of the community may subscribe. It is intended that the list
+be used by the committee, though community members might be asked
+to post there during relevant conversations.
+
+Any business not on the public mailing list will be of a sensitive nature,
+most likely pertaining to individuals. In particular, discussions of selecting
+new members and of violations of the guidelines for respectful communication
+will not appear on the mailing list.
+
+Term Limits and Committee Selection
+-----------------------------------
+
+Any community thrives best by continuing to refresh itself with new members.
+Accordingly, membership on the committee comes with a three-year term. Terms
+begin/end in October. Toward the end of an individual's term, the secretary of
```

Comment:
```suggestion
Accordingly, membership on the committee comes with a three-year term. 
Toward the end of an individual's term, the secretary of
```

It seems that later we say we call for nominations when we drop below 9. Doesn’t that naturally lead to more organic start/end of terms?

Reply:
That's a good question. I like the idea of spacing out when terms expire, so that we have a regular cadence of comings and goings. If someone resigns early, does their replacement get a 3-year term when they arrive? or do they fill out the remainder of the resigner's term? It's not clear to me what's best.

Reply:
Seems clear to me: a full three year term. Why should a new member be treated differently based on the event that triggered the nomination…

Reply:
Because staggering ends of terms has a countervailing benefit to treating members equally without regard to when they step in.

I honestly don't feel strongly -- and you seem to -- but I'd like one other voice to speak up on this point before updating the text.

Reply:
> Because staggering ends of terms has a countervailing benefit to treating members equally without regard to when they step in.

Sorry, I don’t understand what you are saying

Reply:
There is a tension between two nice attributes of a the committee:

1. Terms end in a staggered fashion, with roughly 1/3 of the committee expiring every year, on a predictable schedule.
2. Every member selected to the committee is granted a 3-year term, regardless of the conditions around their nomination.

Both of these are nice things to have. However, I think we cannot have both. In having members who replace early resigners fill out the unfinished term, we keep property (1). In having all members start with a 3-year term, we keep property (2), but we may slowly lose (1), depending on when people resign.

I originally phrased the bylaws to implicitly favor (1), whereas you are advocating for (2).

Does this clarify?

Reply:
I agree with Ioachim. A full three-year term sounds reasonable. It's relatively cheap to track ends of terms.

Reply:
Yes, I don’t really care about (1). I expect that a fair fraction of terms will expire earlier than three years (people losing interest…). And because of our “accept many, re-nominate when <10” rule, even (1) might not lead to predictable schedule. So when I have to choose between a (maybe) predictable schedule, and treating all new members equally, I strongly go for the latter :)